osu.Game/Rulesets/Mods/ModHardRock.cs

```diff
@@ -2,16 +2,26 @@
 // Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
 
 using System;
+using osu.Game.Beatmaps;
 using osu.Game.Graphics;
 
 namespace osu.Game.Rulesets.Mods
 {
-    public abstract class ModHardRock : Mod
+    public abstract class ModHardRock : Mod, IApplicableToDifficulty
     {
         public override string Name => "Hard Rock";
         public override FontAwesome Icon => FontAwesome.fa_osu_mod_hardrock;
         public override ModType Type => ModType.DifficultyIncrease;
         public override string Description => "Everything just got a bit harder...";
         public override Type[] IncompatibleMods => new[] { typeof(ModEasy) };
+
+        public void ApplyToDifficulty(BeatmapDifficulty difficulty)
+        {
+            const float ratio = 1.4f;
+            difficulty.CircleSize *= ratio;

```

Comment:
Ah, I saw this but thought it was specific to the cursor size adjustment. Should probably be changes, yes.